augur/curate/titlecase.py

```diff
@@ -98,7 +98,7 @@ def run(args, records):
 
             titlecased_string = titlecase(record[field], articles, abbreviations)
         
-            failure_message = f"Failed to titlecase {field!r}:{record.get(field)!r} in record {record_id!r}"
+            failure_message = f"Failed to titlecase {field!r}:{record.get(field)!r} in record {record_id!r} because the value is a {type(record.get(field))!r} and is not a string."
```

Comment:
Using `type(...).__name__` will print the class name without the `<class ... >` bit in the message.

Reply:
Thanks! I'll try it

Reply:
After [some reading](https://stackoverflow.com/questions/510972/getting-the-class-name-of-an-instance), [fixed](https://github.com/nextstrain/augur/pull/1197/commits/24fe373dd003eb5671b2e1b0c19a04c1185c06c7) and tested. Thanks!